cpp/ps0001/ps0001cpp.cpp

```diff
@@ -1,21 +1,17 @@
-/******************************************************************************
-
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
 #include <iostream>
 
 using namespace std;
 int main()
 {
-    int a,b,c,result;
-    cout <<"enter three numbers to add \n";
-    cin>>a>>b>>c;
-    result=a+b+c;
-    cout<<"the sum of a , b and c =" << result ;
+    // prompting users for 3 numbers  1/2
+    unsigned short a, b, c, result;
+    cout << "enter three numbers to add \n";
+    cin >> a >> b >> c;
+
```

Comment:
- [x] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add